### ISSUE TITLE
Empty array in pass state parameters

### ISSUE THREAD

**Issue by redvex** [nibble-4bits/aws-local-stepfunctions#81]
When I try to return an array from a pass state parameters, the convert it to an object.

## Step to reproduce

```
import { StateMachine } from 'aws-local-stepfunctions';

const machineDefinition = JSON.parse(JSON.stringify({
  StartAt: 'FilterInput',
  States: {
    FilterInput: {
      Type: 'Pass',
      Parameters: [{
        foo: 'bar'
      }], 
      // Result: [{
      //   foo: 'bar'
      // }],
      End: true
    }
  },
}));

test('OK', async () => {
  // GIVEN
  const myInput = {};

  // WHEN
  const stateMachine = new StateMachine(machineDefinition);
  const execution = stateMachine.run(myInput);
  const result = await execution.result;

  // THEN
  expect(result).toStrictEqual([{foo: 'bar'}]);
});
```

## Desired outcome

The state machine should return the array

## Error

```
expect(received).toStrictEqual(expected) // deep equality

    Expected: [{"foo": "bar"}]
    Received: {"0": {"foo": "bar"}}
```

## Workaround

If I set the empty array in the `Result`, it works, but it's not a viable solution.

**Comment by nibble-4bits** [comment]
The spec mentions that the _Payload Template_ of the `Parameters` field must be a JSON object, however it seems the AWS implementation does in fact allow having an array value for the `Parameters` field, so I'll label this as an enhancement.

---

As a workaround, you could also do something like this:

```js
const result = await execution.result;
const array = Object.values(result);
```

**Comment by nibble-4bits** [comment]
This feature has been released in [v1.3.0](https://github.com/nibble-4bits/aws-local-stepfunctions/releases/tag/v1.3.0)
